python/lsst/afw/detection/footprint.cc

```diff
@@ -86,8 +86,13 @@ PYBIND11_PLUGIN(_footprint) {
 
     clsFootprint.def("assign", &Footprint::operator=);
 
-    // No good way to wrap the bitmask, not sure if it is needed
-    clsFootprint.def("intersectMask", [](Footprint & self, image::Mask<image::MaskPixel> const & mask) { return self.intersectMask(mask); });
+    // Default argument for bitmask doesn't work here.
+    clsFootprint.def("intersectMask", [](Footprint &self, image::Mask<image::MaskPixel> const &mask) {
+        return self.intersectMask(mask);
+    });
+    clsFootprint.def("intersectMask",
+                     [](Footprint &self, image::Mask<image::MaskPixel> const &mask,
+                        image::MaskPixel bitmask) { return self.intersectMask(mask, bitmask); });
```

Comment:
Please consider wrapping the remaining methods (and reporting the lack of unit tests)